### ISSUE TITLE
"Abstract/Zusammenfassung" in the bookmark does not index the contents in document correctly.

### ISSUE THREAD

**Issue by XiaodiJin** [tud-cd/tudscr#30]
I write my thesis in English in German. So I should write one Germany abstract and then one English version. I used 

`\TUDoption{abstract}{section,tocmultiple}` 

before the abstract contents, in order to make the index of abstract in toc and bookmark.
The **Zusammengfassung** and **Abstract** in toc can index the contents correctly, but **Zusammengfassung/Abstract in bookmark will index the title page rather than abstract page of the documents**, which is wrong. How should I fix it?  Thanks advance.

The code is simplified attach in the **_test.zip_** in **_test_Abstractinbookmark.tex_**  and the produced datei is **_test_Abstractinbookmark.pdf_**. You can see in the bookmark of **_test_Abstractinbookmark.pdf_**, wenn you touch the **Zusammengfassung/Abstract** in bookmark, it index the **_page 1 title page_** rather than **_page 2 abstract_**. 

I use Texstudio as the interface, miktex is the distribution, plattform is windos10. 
[test.zip](https://github.com/tud-cd/tudscr/files/1744840/test.zip)

**Comment by XiaodiJin** [comment]
Besides, the **_Zusammengfassung/Abstract_** of TOC index to the **_page 1 title page_** rather than **_page 2 abstract_**, too.

**Comment by mrpiggi** [comment]
Congratulations, you found a bug ;) The code, which actually should work, looks like this:
```latex
\documentclass[english,cd=nocolor]{tudscrreprt}
\usepackage[T1]{fontenc}
\usepackage{selinput}
\SelectInputMappings{adieresis={ä},germandbls={ß}}

\usepackage[ngerman,main=english]{babel}
\usepackage{bookmark}
\usepackage{blindtext}

\begin{document}
\pagenumbering{roman}
\title{text}
\maketitle

\TUDoption{abstract}{section,tocmultiple}%
\begin{abstract}[ngerman,pagestyle=plain]
\blindtext
\nextabstract[english]
\blindtext
\end{abstract}

% Inhaltsverzeichnis
\tableofcontents

\cleardoubleemptypage
\pagenumbering{arabic}

\chapter{ Introduction }
\par This is the introduction.
\section{ Motivation }
\par This is motivation.
\section{ Thesis outline }
\par This is outline.
\end{document}
```
But as you explained, the hyperlink points to the wrong page (because of a missing anchor, which isn't set but needed). A possible workaround would be to suppress the automatically created toc entry and create one manually by yourself.

```latex
\documentclass[english,cd=nocolor]{tudscrreprt}
\usepackage[T1]{fontenc}
\usepackage{selinput}
\SelectInputMappings{adieresis={ä},germandbls={ß}}

\usepackage[ngerman,main=english]{babel}
\usepackage{bookmark}
\usepackage{blindtext}

\begin{document}
\pagenumbering{roman}
\title{text}
\maketitle

\TUDoption{abstract}{section,multiple,notoc}%
\newcommand*\tempheading{}
\begingroup
\begin{otherlanguage}{ngerman}
\xdef\tempheading{\abstractname/}
\end{otherlanguage}
\xdef\tempheading{\tempheading\abstractname}
\endgroup

\begin{abstract}[ngerman,pagestyle=plain]
\addchaptertocentry{}{\tempheading}
\blindtext
\nextabstract[english]
\blindtext
\end{abstract}

% Inhaltsverzeichnis
\tableofcontents

\cleardoubleemptypage
\pagenumbering{arabic}

\chapter{ Introduction }
\par This is the introduction.
\section{ Motivation }
\par This is motivation.
\section{ Thesis outline }
\par This is outline.
\end{document}
```

Cheers

**Comment by XiaodiJin** [comment]
@mrpiggi Thanks for the answer. If this is a bug, how should I post it to the auther?
I have tried your code. But it produces also two "Zusammenfassung/Abstract" in the bookmark, and the first points to the Title page and the second points to the right page. Please see the figure.
![image](https://user-images.githubusercontent.com/30651919/36631398-71b799f4-1977-11e8-877f-96cf21132cc1.png)

**Comment by mrpiggi** [comment]
I'm the author, the bug is fixed for a future release. Sorry for the provided code, there was a little mistake. This should work:
```latex
\documentclass[english,cd=nocolor]{tudscrreprt}
\usepackage[T1]{fontenc}
\usepackage{selinput}
\SelectInputMappings{adieresis={ä},germandbls={ß}}

\usepackage[ngerman,main=english]{babel}
\usepackage{bookmark}
\usepackage{blindtext}

\begin{document}
\pagenumbering{roman}
\title{text}
\maketitle


\TUDoption{abstract}{section,multiple,notoc}%
\newcommand*\tempheading{}
\begingroup
\begin{otherlanguage}{ngerman}
\xdef\tempheading{\abstractname/}
\end{otherlanguage}
\xdef\tempheading{\tempheading\abstractname}
\endgroup

\phantomsection\addchaptertocentry{}{\tempheading}
\begin{abstract}[ngerman,pagestyle=plain]
\blindtext
\nextabstract[english]
\blindtext
\end{abstract}

% Inhaltsverzeichnis
\tableofcontents

\cleardoubleemptypage
\pagenumbering{arabic}

\chapter{ Introduction }
\par This is the introduction.
\section{ Motivation }
\par This is motivation.
\section{ Thesis outline }
\par This is outline.
\end{document}
```

**Comment by mrpiggi** [comment]
c16950ac07ff80acd89e5b02eac52fe4e53961dc closes #30

**Comment by mrpiggi** [comment]
c16950ac07ff80acd89e5b02eac52fe4e53961dc fixes #30
